bindings/python/.evergreen/test.sh

```diff
@@ -5,20 +5,64 @@
 set -o xtrace   # Write all commands first to stderr
 set -o errexit  # Exit the script with error if any of the commands fail
 
-# This library is built by libmongocrypt/.evergreen/compile.sh
+# libmongocrypt is built by libmongocrypt/.evergreen/compile.sh
 evergreen_root="$(cd ../../../; pwd)"
-export PYMONGOCRYPT_LIB=${evergreen_root}/install/libmongocrypt/lib64/libmongocrypt.so
 
-PYTHONS=("/opt/python/2.7/bin/python" \
-         "/opt/python/3.4/bin/python3" \
-         "/opt/python/3.5/bin/python3" \
-         "/opt/python/3.6/bin/python3" \
-#         "/opt/python/pypy/bin/pypy" \  # TODO: PyPy segfaults on RHEL 6.2
-         "/opt/python/pypy3.6/bin/pypy3")
+if [ "Windows_NT" = "$OS" ]; then # Magic variable in cygwin
+    PYMONGOCRYPT_LIB=${evergreen_root}/install/libmongocrypt/bin/mongocrypt.dll
+    export PYMONGOCRYPT_LIB=$(cygpath -m $PYMONGOCRYPT_LIB)
+    # We need to create virtualenvs to install pymongocrypt's
+    # cffi, cryptography, and pymongo dependencies correctly from
+    # wheels. Otherwise, setup.py attempts and fails to install
+    # from the source distributions (.tar.gz).
+    /cygdrive/c/python/Python27/python -m virtualenv venv27
+    PYTHON27="$(pwd)/venv27/Scripts/python"
+    $PYTHON27 -m pip install . pymongo
+
+    /cygdrive/c/python/Python36/python -m virtualenv venv36
+    PYTHON36="$(pwd)/venv36/Scripts/python"
+    $PYTHON36 -m pip install . pymongo
+
+    # 3.4 supports '-m venv', not '-m virtualenv'.
+#    /cygdrive/c/python/Python34/python -m venv venv34
+#    PYTHON34="$(pwd)/venv34/Scripts/python"
+#    $PYTHON34 -m pip install --upgrade pip
```

Comment:
Why are you trying to upgrade pip?

Reply:
The version of pip is too old to download and install cffi+cryptography correctly. I believe it was version 8 or 9.

Reply:
OK, I'll look into this as a separate task.